docs/html/reference/pip_install.rst

```diff
@@ -72,7 +72,7 @@ Installation Order
 ++++++++++++++++++
 
 As of v6.1.0, pip installs dependencies before their dependents, i.e. in
-"topological order".  This is the only commitment pip currently makes related
+"topological order."  This is the only commitment pip currently makes related
```

Comment:
I think the older form is correct. Any reasons to change this?

Reply:
The rule in English (at least in the US), is that commas and periods go inside the end of quotation marks, and colons and semi-colons outside. You can verify this by looking in any book, magazine, or newspaper (e.g. the New York Times). It's possible British English is more relaxed and allows both.

I do think the document uses quotes around too many words. But I left all of those alone and only changed the things I thought were wrong.

Reply:
The older form is fine in British English but would require the change in American English according to https://en.wikipedia.org/wiki/Quotation_marks_in_English#Order_of_punctuation :)

I guess the doc is in american english and the changes are therefore fine (but still feel wrong for me as french punctuation rules are more similar to british ones).